Legacy/bonej/src/main/java/org/bonej/plugins/ParticleAnalysis.java

```diff
@@ -655,9 +655,11 @@ static ImagePlus getBinaryParticle(final int p, final ImagePlus imp, final int[]
 					point.y += yOffset;
 					point.z += zOffset;
 				}
-				surfacePoints.add(points);
 				if (points.isEmpty()) {
 					IJ.log("Particle " + p + " resulted in 0 surface points");
+					surfacePoints.add(null);
```

Comment:
If we don't actually need to differentiate between `null` and `empty` lists (see below), then I'm not a fan of having `null` lists in the List<List<Point3f>>

Reply:
>  I'm not a fan of having `null` lists in the List<List>

Agree. Will avoid in the future. Better to have empty `Lists` and to detect them with `isEmpty()`.